app/assets/javascripts/govuk_publishing_components/lib/govspeak/govspeak-track-links.js

```diff
@@ -0,0 +1,30 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (GOVUK) {
+  function TrackLinks () { }
+
+  TrackLinks.prototype.start = function (scope) {
+    var _this = this
+    var category = scope.getAttribute('data-track-links-category')
+
+    scope.querySelectorAll('a').forEach(function (link) {
```

Comment:
[`forEach` on a NodeList isn't supported by IE](https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach#browser_compatibility) - I think this might need to be replaced by a `for` loop.